src/utils/utils.ts

```diff
@@ -65,6 +68,31 @@ export const authApi = async (): Promise<IAuthCredentials | undefined> => {
   }
 };
 
+export const authenticatedFetch = async (
```

Comment:
you are still going to need the rest of the fetches parameters because of things like method and body that other parts of the codebase is using.

Reply:
Comments please :)